addons/point_of_sale/models/pos_order.py

```diff
@@ -1171,7 +1171,7 @@ def _compute_amount_line_all(self):
         return {
             'price_subtotal_incl': taxes['total_included'],
             'price_subtotal': taxes['total_excluded'],
-            'taxes': taxes['taxes']
+            'tax_ids': taxes['taxes']
```

Comment:
compute_all() doesn't return a recordset of taxes, but a list of dicts with the tax details. So even with this change, it would be incorrect and still crash.

In my opinion, this row shouldn't be there and can be removed completely. 
The taxes are already set on the line here, as we are using them to calculate that amounts.

```suggestion
```